django_project/core/urls.py

```diff
@@ -23,6 +23,7 @@
     url(r'^accounts/', include('allauth.urls')),
     url(r'^', include('bims.urls')),
     url(r'^api-auth/', include('rest_framework.urls')),
+    url('^contact/', include('contactus.urls')),
```

Comment:
Where did this come from?

Reply:
Okay @dimasciput am fixing these and the urls are coming from the third party app for contact us.
Thanks